src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -152,12 +160,70 @@ export default class FiatConnectQuote extends NormalizedQuote {
     return this.quoteResponseKycSchema?.kycSchema
   }
 
-  // TODO: Dynamically generate time estimation strings
-  getTimeEstimation(): SettlementTime {
-    // payment method can only be bank or fc mobile money
-    return this.getPaymentMethod() === PaymentMethod.Bank
-      ? SettlementTime.ONE_TO_THREE_DAYS
-      : SettlementTime.LESS_THAN_24_HOURS
+  getTimeEstimation(): SettlementEstimation {
+    const fiatAccountInfo = this.quote.fiatAccount[this.getFiatAccountType()]
+    if (!fiatAccountInfo?.settlementTimeUpperBound) {
+      // payment method can only be bank or fc mobile money
+      return this.getPaymentMethod() === PaymentMethod.Bank
+        ? DEFAULT_BANK_SETTLEMENT_ESTIMATION
+        : DEFAULT_MOBILE_MONEY_SETTLEMENT_ESTIMATION
+    }
+
+    const daysToSettlementEstimation = (
+      lowerBound: number,
+      upperBound: number
```

Comment:
would be nice if these parameter names included units

Reply:
I'd probably opt for destructured assignment too, to prevent bugs where the order of inputs are swapped. it's an intuitive ordering as is, but it wouldn't hurt.

Reply:
Good call, added both those suggestions